data/json/items/containers.json

```diff
@@ -1664,16 +1664,40 @@
         "description": "This waterproof phone case has a drawing on it that makes it look like an old English police call box.",
         "weight": 1
       },
+      {
+        "id": "black_smart_phone_case",
+        "name": { "str_sp": "black waterproof smartphone case" },
+        "description": "A <black> waterproof case designed for a standard smartphone.  Will protect your smartphone from water damage entirely.",
```

Comment:
Apologies, I didn't mean for you to include the `<>` for these colors, I was just trying to generalize all colors with `<color>`

Reply:
Ha I was like I wonder if that will show it off in the color listed.

Reply:
When I saw your update, I had to go check to make sure I wasn't missing a feature.  It's hard to tell sometimes what is implemented. Sorry for the confusion!